### DISCUSSION TITLE
Form input fields

### DISCUSSION THREAD

**Discussion by fluzzywuzzy** [dompdf/dompdf#2387] [Q&A] [answered]
I am considering some different libraries for creating dynamic PDF's, but there's one feature that most seem to be missing. I just want to ask and make sure that it's missing, or be corrected if it's actually available.

Is it possible to create form fields with this library, so that a user can input text while viewing the PDF in the browser?

All other libraries I have found so far, is either missing this alltogether, or only allows input into fields from a static, allready created, pdf.

Thanks in advance if anyone has the definite answer to this question for me! :)

**Comment by bsweeney** [comment, marked as answer]
Dompdf itself can't add fillable form fields, there's a long-standing issue requesting this feature (#4). If you were using the PDFLib backend you could add form elements directly, but it does complicate the process since you have to manually layout everything.
